extensions/wikia/EditPageLayout/js/plugins/PageControls.js

```diff
@@ -76,7 +77,33 @@
 			} ) );
 
 			// attach events
-			$( '#wpPreview' ).bind( 'click', this.proxy( this.onPreview ) );
+			$( '#wpPreview' ).bind(
+				'click', this.proxy( this.onPreview )
+			).popover({
+				placement: 'top',
+				content: $.msg('editpagelayout-preview-label-desktop'),
```

Comment:
If you're using `$.msg` instead of the `mediawiki.message` library, make sure this is appropriately escaped either here, or where the `content` option is output. Escaping i18n messages is still required, and our older `$.msg` doesn't do this like `mediawiki.message('message').escaped()` does.